src/components/Nav/NavbarButtonGroup.tsx

```diff
@@ -0,0 +1,20 @@
+import { Link } from "react-router-dom";
+
+interface NavbarButtonGroup {
```

Comment:
You have two different names in this component. Is this the `ButtonGroup` or the `NavbarIcon` component?